h2olog.cc

```diff
@@ -81,6 +81,10 @@ static void show_process(pid_t pid)
     }
     size_t nread = fread(cmdline, 1, sizeof(cmdline), f);
     fclose(f);
+    if (nread <= 0) {
```

Comment:
`nread` is `size_t`(unsigned number). So I suppose it is enough to use `== 0`.

Reply:
Exactly. Why don't my C compiler complain about it...

6bf7ca2